judges/quality-of-service/quality-of-service.rb

```diff
@@ -111,4 +111,16 @@
   end
   f.average_pull_hoc_size = hocs.empty? ? 0 : hocs.sum.to_f / hocs.size
   f.average_pull_files_size = files.empty? ? 0 : files.sum.to_f / files.size
+
+  # Average review time
+  review_times = []
+  Fbe.unmask_repos.each do |repo|
+    Fbe.octo.search_issues(
+      "repo:#{repo} type:pr created:>#{f.since.utc.iso8601[0..9]}"
```

Comment:
`f.since.utc.iso8601[0..9]` expression scares me. I know it's used everywhere across the project. But maybe we should not spread it further.

Reply:
@tank-bohr Yes, you may be right and I have suggested a clearer version [here](https://github.com/zerocracy/judges-action/issues/244#issue-2454293095)
But I didn't receive an affirmative answer from @yegor256
If you refer to the [GitHub API reference information](https://docs.github.com/en/search-github/getting-started-with-searching-on-github/understanding-the-search-syntax#query-for-dates) on constructing a search query for a date, then it indicates that they use the ISO8601 format and this line takes on a more understandable meaning.

```ruby
# for date only
Time.now.utc.iso8601[0..9]
"2024-08-28T08:11:08Z"
 ^^^^^^^^^^

# for date and time
Time.now.utc.iso8601[0..18]
"2024-08-28T08:13:24Z" 
 ^^^^^^^^^^^^^^^^^^^
```